tests/integration_tests/test_storage_migration.py

```diff
@@ -59,3 +95,19 @@ def test_that_storage_matches(tmp_path, source_root, snapshot, monkeypatch):
             snapshot.assert_match(
                 str(experiment.response_configuration) + "\n", "responses"
             )
+
+            summary_data = ensemble.load_responses(
+                "summary",
+                tuple(ensemble.get_realization_list_with_responses("summary")),
+            )
+            snapshot.assert_match(
+                summary_data.to_dataframe().astype(np.float32).to_csv(),
+                "summary_data",
+            )
+            snapshot.assert_match_dir(
```

Comment:
I think this should be made somehow to not enforce the file structure, but just that the contents of the observations are parsed in correctly. Enforcing the directory structure of observations to be the same will make it impossible to change it away from one observation `.nc` file per observation specification, which currently slows down things quite a bit

Reply:
It only enforces that the dictionary remains the same. Any change to this can just fix up the snapshot. It does not match against any `.nc` files just the values in the dictionary.